tests/datavolume_test.go

```diff
@@ -119,6 +120,84 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 		)
 	})
 
+	Describe("Verify DataVolume with block mode", func() {
+		var err error
+		var storageClass *storageV1.StorageClass
+		var pod *v1.Pod
+		var pv *v1.PersistentVolume
+
+		BeforeEach(func() {
+			By(fmt.Sprintf("Creating storageClass for Block PV"))
+			storageClass, err = f.CreateStorageClassFromDefinition(utils.NewStorageClassForBlockPVDefinition("manual"))
+			Expect(err).ToNot(HaveOccurred())
+
+			pod, err = utils.FindPodByPrefix(f.K8sClient, "cdi", "cdi-block-device", "kubevirt.io=cdi-block-device")
```

Comment:
If you make functest-block-device a DaemonSet I don't think you will have to worry about node affinity at all

Reply:
This is good advice, I would do that to make thing simpler for the test.